lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -205,6 +205,17 @@ def initialize(connection, logger, conn_params, config)
           version_num = 202
         end
         @crdb_version = version_num
+
+        begin
+          query_value("SELECT crdb_internal.increment_feature_counter('activerecord v6.1')")
```

Comment:
Can I get the Rails version number programatically here?

Reply:
/chime 

i am wondering if this creates an extra undesired round trip on each ORM. maybe this should be part of the connection parameters instead :\

Reply:
Good point, though I'm not sure that passing as a connection parameter would completely solve the problem since it would be rejected by earlier CRDB versions. You could then retry and omit the parameter, but then you haven't saved a round-trip.

Reply:
Hmm, though another nice thing about using a connection parameter is I think we could get away with not hashing the value before reporting. I'll think on this.

Reply:
Do we ignore or hard fail when we find a conn param we don't like?

Otherwise, I think most ORMs I think have a stanza where they send a bunch of initial sql commands up to pg catalog and the like, it'd be ideal if we tacked this on to one of them.

Reply:
Passing additional fields into `conn_params` won't raise an error.

It looks like those startup queries are run every time a connection pool is created through `establish_connection`, and they're made when we call the parent method here.

Also, `ActiveRecord::VERSION::STRING` will return the current version.

Reply:
CockroachDB will error out on an unrecognized connection parameter.

I updated my code to spawn a separate thread for this query since there's no reason to block on it, so hopefully this avoids any latency hit. Also incorporated `ActiveRecord::VERSION`, thanks for the pointer.